dev-docs/RFCs/proposals/pydeck-interactive-api-rfc.md

```diff
@@ -32,21 +32,34 @@ Question: Should we use ipywidgets style API, or deck.gl API, or transport API o
 
 ## Use Cases
 
-### "Raw" Events (Click and Hover, etc)
+### Events (Click and Hover, etc)
 
-Hover
+**Hover Tooltip**
 - Currently pydeck has "partial support" for Hover events through the [`tooltip` prop](https://deckgl.readthedocs.io/en/latest/tooltip.html).
-- This is somewhat "raw" ("innerHTML" props), but serves its purpose.]
-- An alternate (or parallel) onHover handler would be a P2
+- The API is somewhat "raw" ("innerHTML" props), but serves its purpose.
 
-Click
+**Hover**
+- An basic onHover callback
```

Comment:
I imagine that onHover could change other panes within a Jupyter environment, like this plotly example but in the reverse direction–a user can interact with the map (e.g. brushing over an area) and it modifies the chart:

<img src="https://user-images.githubusercontent.com/1280389/27838299-add56ade-60b7-11e7-82e3-7ad5ae7c489d.gif"></img>